pkg/storaged/nfs-details.jsx

```diff
@@ -284,27 +284,27 @@ export class NFSDetails extends React.Component {
 
         const header = (
             <Card>
-                <CardHeader>
+                <CardHeader actions={{
+                    actions: <>
+                        { entry.mounted
+                            ? <StorageButton onClick={unmount}>{_("Unmount")}</StorageButton>
+                            : <StorageButton onClick={mount}>{_("Mount")}</StorageButton>
+                        }
+                        { "\n" }
+                        { entry.fstab
+                            ? [
+                                <StorageButton key="1" onClick={edit}>{_("Edit")}</StorageButton>,
+                                "\n",
+                                <StorageButton key="2" onClick={remove} kind="danger">{_("Remove")}</StorageButton>
+                            ]
+                            : null
```

Comment:
This added line is not executed by any test.  [Details](https://cockpit-logs.us-east-1.linodeobjects.com/pull-18672-20230426-112552-0a013b1d-fedora-37-devel//Coverage/lcov/github-pr.diff.gcov.html)